employees/models.py

```diff
@@ -1,9 +1,11 @@
+import uuid
+
 from django.db import models
+from django_extensions.db.models import TimeStampedModel
 
 
-class BaseModel(models.Model):
-    created_date = models.DateTimeField(auto_now_add=True)
-    modified_date = models.DateTimeField(auto_now=True)
+class BaseModel(TimeStampedModel):
+    id = models.CharField(max_length=50, primary_key=True, default=uuid.uuid4())
```

Comment:
consider adding 
"config = JSONField(default=dict, blank=True)"
that'll help us to add extra data into an object if needed. like temp data.